frontend/src/App.tsx

```diff
@@ -10,8 +10,11 @@ import Signup from './pages/Signup';
 import Navbar from './components/Navbar';
 
 import ProfilePage from './pages/ProfilePage';
+import { useTheme } from '@material-ui/core/styles';
 
 function App() {
+
+  const theme = useTheme();
```

Comment:
Was this actually necessary here? Or is it something we need in each component?

Reply:
Okay, I committed the new change. I tested it locally, and it does not require useTheme.